orchard_group_hash.py

```diff
@@ -136,21 +136,17 @@ def group_hash(d, m):
 
 
 def main():
-    test_vectors = [
-        (b"z.cash:test", b"Trans rights now!"),
+    group_hash_test_vectors = [
+        # This is the Pallas test vector from the Sage and Rust code (in affine coordinates).
+        (b"z.cash:test", b"Trans rights now!", Point(Fp(10899331951394555178876036573383466686793225972744812919361819919497009261523),
+                                                     Fp(851679174277466283220362715537906858808436854303373129825287392516025427980))),
```

Comment:
This refactor interferes with the test vector generator. We are supposed to be _generating_ test vectors here, not checking them. I special-cased the one from the Sage code, but the intention is that we'll use randomness to generate test vectors for which this kind of check would be pointlessly tautological.